ui/desktop/electron-app/src/preload.js

```diff
@@ -61,3 +61,11 @@ process.once('loaded', () => {
     }
   });
 });
+
+contextBridge.exposeInMainWorld('electron', {
+  ipcRenderer: {
```

Comment:
We shouldn't expose the bare ipcRenderer as that can be a [security concern](https://www.electronjs.org/docs/latest/tutorial/context-isolation#security-considerations).